### DISCUSSION TITLE
ElevenClock Showing Behind DisplayFusion Taskbar

### DISCUSSION THREAD

**Discussion by Zevgor** [martinet101/ElevenClock#839] [General]
Hello,
I wanted to ask if there was a way to force ElevenClock to bring the widget to the front.
I use DisplayFusion to have a 2nd monitor taskbar, and ElevenClock displays behind it, rather than on top/overlaying it.
See here:

![image](https://user-images.githubusercontent.com/12678166/180284569-90b21043-a6c2-4c98-953b-87ab21587f6f.png)

You will see the faint ElevenClock behind the regular clock.

**Comment by marticliment** [comment]
True. Can you please send me your ElevenClock log?

**Comment by dannylussier** [comment]
Sorry for hijacking this, but I have the same issue. Should I post my log or DM it?

**Reply by marticliment** [reply]
I have received it

**Comment by marticliment** [comment]
You can post it here if you want, there shouldn't be sensitive info. However, if you prefer, you can send it to me at marticlilop@gmail.com

**Comment by marticliment** [comment]
Hi guys,

could you please try to run ElevenClock as administrator?
